src/mam4xx/tropopause.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef MAM4XX_TROPOPAUSE_HPP
+#define MAM4XX_TROPOPAUSE_HPP
+
+#include <haero/math.hpp>
+#include <mam4xx/aero_config.hpp>
+
+#include <fstream>
+#include <iostream>
+
+namespace mam4 {
+
+namespace tropopause {
+
+using ConstColumnView = haero::ConstColumnView;
+// From radconstants
+constexpr int nswbands = 14;
+constexpr int nlwbands = 16;
+using View2D = DeviceType::view_2d<Real>;
+constexpr Real km_inv_to_m_inv = 0.001; // 1/km to 1/m
+
+constexpr Real shr_const_rgas =
+    haero::Constants::r_gas * 1e3; // Universal gas constant ~ J/K/kmole
+constexpr Real shr_const_mwdair = haero::Constants::molec_weight_dry_air *
+                                  1e3; // molecular weight dry air ~ kg/kmole
+constexpr Real shr_const_cpdair =
+    haero::Constants::cp_dry_air; // specific heat of dry air   ~ J/kg/K
+
+constexpr Real cnst_kap =
+    (shr_const_rgas / shr_const_mwdair) / shr_const_cpdair; //  ! R/Cp
+
+constexpr Real cnst_faktor =
+    -haero::Constants::gravity /
+    haero::Constants::r_gas_dry_air; // acceleration of gravity ~ m/s^2/Dry air
+                                     // gas constant     ~ J/K/kg
+constexpr Real cnst_ka1 = cnst_kap - 1.0;
```

Comment:
Are the above constants going to be used outside of the `get_dtdz` function? If any of them are specific to that function, they can be placed inside it.

Reply:
They will also be in the function `twmo` in tropopause